src/js/components/MaskedInput/__tests__/MaskedInput-test.js

```diff
@@ -191,6 +191,55 @@ describe('MaskedInput', () => {
     expect(onChange).toHaveReturnedWith('aa!');
   });
 
+  test('should not enable to type beyond options via keyboard', async () => {
+    const onChange = jest.fn((event) => event.target.value);
+    const { getByTestId, container } = render(
+      <MaskedInput
+        data-testid="test-input"
+        id="item"
+        name="item"
+        mask={[
+          {
+            options: ['aaa', 'aba', 'abb'],
+            regexp: /\w$/,
+          },
+        ]}
+        onChange={onChange}
+      />,
+    );
+    expect(container.firstChild).toMatchSnapshot();
+
+    fireEvent.change(getByTestId('test-input'), { target: { value: 'abbb' } });
```

Comment:
Can we use `userEvent` instead of `fireEvent` and `getByRole` instead of `getByTestId`? See https://github.com/grommet/grommet/blob/master/CONTRIBUTING.md#testing-your-code for more info

Reply:
Sure, sorry about that.